### PR TITLE
Fixing error in NIST property library

### PR DESCRIPTION
**Pull request by andrewlee94**
## Summary/Motivation:
The was an error in the NIST library for the calculation of specific molar enthalpy using the Shomate equation.

## Changes proposed in this PR:
- Remove reference temperature terms in Shomate equation for enthalpy
-Update tests and confirmed they match numbers from NIST Webbook
- Removed patch fix in natural gas properties in power generation library

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: idaes/generic_models/properties/core/pure/NIST.py [hunk 1]
```diff
@@ -109,11 +109,11 @@ def return_expression(b, cobj, T):
         H = (cobj.cp_mol_ig_comp_coeff_H if not h_form
              else 0*pyunits.kJ*pyunits.mol**-1)
 
-        h = (cobj.cp_mol_ig_comp_coeff_A*(t-tr) +
```
Comment (AlexNoring) [inline idaes/generic_models/properties/core/pure/NIST.py:112]: `tr` defined above is now unused so you should delete it too

**Review by AlexNoring — CHANGES_REQUESTED** [review]
tr is defined and unused, but other than that it looks good

**Review by AlexNoring — APPROVED** [review]
lgtm

**Review by agarciadiego — APPROVED** [review]
LGTM

**Pull request merged**